module1/projects/dmv/iteration_1.md

```diff
@@ -15,12 +15,13 @@ You should explore:
 * Take note of the project folder structure. Is there anything you haven't seen before?
 
 ### Debugging
-If you haven't already, run the tests now. You should see some failing tests. Work through fixing the code until the tests pass. You only change the application code, not the tests. It might be helpful to run each test one at a time and fix one by one.
+If you haven't already, run the tests now. You should see some failing tests. Work through fixing the code until the tests pass. You will only change the application code in `spec/dmv_spec.rb`, not the tests. You will make updates to the tests in `spec/dmv_data_service_spec.rb`, not the application code. It might be helpful to run each test one at a time and fix one by one.
```

Comment:
"You will only change the application code in `spec/dmv_spec.rb`, not the tests."  is a bit contradictory. dmv_spec is a test, not application code.